File: app/src/main/java/com/goranch/publicapis/api/ShazamService.java
package com.goranch.publicapis.api;

import com.goranch.publicapis.api.model.shazam.ApiTopLevelObject;

import io.reactivex.Observable;
import retrofit2.http.GET;

public interface ShazamService {

    @GET("/shazam/v2/en/GB/android/-/chart/recent-music/1month/-/-")
    Observable<ApiTopLevelObject> getTrending();
}

File: app/src/main/java/com/goranch/publicapis/api/ShazamService.kt
package com.goranch.publicapis.api

import com.goranch.publicapis.api.model.shazam.ApiTopLevelObject

import io.reactivex.Observable
import retrofit2.http.GET

interface ShazamService {

    @get:GET("/shazam/v2/en/GB/android/-/chart/recent-music/1month/-/-")
    val trending: Observable<ApiTopLevelObject>
}
